libs/providers/flagd/src/lib/flagd-provider.spec.ts

```diff
@@ -245,54 +249,46 @@ describe(FlagdProvider.name, () => {
     });
 
     describe(FlagdProvider.prototype.resolveBooleanEvaluation.name, () => {
-      it(`should call ${ServiceClient.prototype.resolveBoolean} with key and context and return details`, async () => {
+      it(`should call ${ServiceClient.prototype.resolveBoolean} with key and context and return falsy value`, async () => {
         const val = await client.getBooleanDetails(BOOLEAN_KEY, false, TEST_CONTEXT);
         expect(basicServiceClientMock.resolveBoolean).toHaveBeenCalledWith({
           flagKey: BOOLEAN_KEY,
           context: TEST_CONTEXT,
         }, expect.any(Function));
         expect(val.value).toEqual(false);
-        expect(val.variant).toEqual(BOOLEAN_VARIANT);
-        expect(val.reason).toEqual(REASON);
       });
     });
 
     describe(FlagdProvider.prototype.resolveStringEvaluation.name, () => {
-      it(`should call ${ServiceClient.prototype.resolveString} with key and context and return details`, async () => {
+      it(`should call ${ServiceClient.prototype.resolveString} with key and context and return falsy value`, async () => {
         const val = await client.getStringDetails(STRING_KEY, '', TEST_CONTEXT);
         expect(basicServiceClientMock.resolveString).toHaveBeenCalledWith({
           flagKey: STRING_KEY,
           context: TEST_CONTEXT,
         }, expect.any(Function));
         expect(val.value).toEqual('');
-        expect(val.variant).toEqual(STRING_VARIANT);
-        expect(val.reason).toEqual(REASON);
       });
     });
 
     describe(FlagdProvider.prototype.resolveNumberEvaluation.name, () => {
-      it(`should call ${ServiceClient.prototype.resolveFloat} with key and context and return details`, async () => {
+      it(`should call ${ServiceClient.prototype.resolveFloat} with key and context and return falsy value`, async () => {
         const val = await client.getNumberDetails(NUMBER_KEY, 0, TEST_CONTEXT);
         expect(basicServiceClientMock.resolveFloat).toHaveBeenCalledWith({
           flagKey: NUMBER_KEY,
           context: TEST_CONTEXT,
         }, expect.any(Function));
         expect(val.value).toEqual(0);
-        expect(val.variant).toEqual(NUMBER_VARIANT);
-        expect(val.reason).toEqual(REASON);
```

Comment:
IMHO these tests were asserting too much. These ones are just about proper zero-values, and the other assertions are covered by the tests above.